pkg/trie/triedb/codec/key_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package codec
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func repeatBytes(b byte, n int) (slice []byte) {
+	slice = make([]byte, n)
+	for i := range slice {
+		slice[i] = b
+	}
+	return slice
```

Comment:
use `bytes.Repeat` https://pkg.go.dev/bytes#Repeat
```suggestion
	return bytes.Repeat([]byte{b}, n)
```

Reply:
Good catch! very good opportunity to improve our tests! thks